tensorflow/lite/tools/benchmark/experimental/delegate_performance/android/src/main/native/latency_benchmark.cc

```diff
@@ -157,6 +157,10 @@ std::vector<std::string> ParseArgumentsFromTfLiteSettings(
             absl::StrFormat("--num_threads=%d",
                             tflite_settings.xnnpack_settings()->num_threads()));
       }
+      if (tflite_settings.xnnpack_settings() &&
+          tflite_settings.xnnpack_settings()->flags() | TFLITE_XNNPACK_DELEGATE_FLAG_FORCE_FP16) {
```

Comment:
needs to be changed to:
```
tflite_settings.xnnpack_settings()->flags() ==
              XNNPackFlags_TFLITE_XNNPACK_DELEGATE_FLAG_FORCE_FP16
```

Reply:
Fixed at [de92054].(https://github.com/tensorflow/tensorflow/pull/62591/commits/de92054e43f01478d2680c4ad34199d2fb15ca0d)

Reply:
Sorry what i meant is to move `if (tflite_settings.xnnpack_settings())` as the outer condition, and nest both `if (tflite_settings.xnnpack_settings()->num_threads()) {}` and `if (tflite_settings.xnnpack_settings()->flags() == XNNPackFlags_TFLITE_XNNPACK_DELEGATE_FLAG_FORCE_FP16) {}` within it.

Thanks!

Reply:
Okay. Fixed [7c30388](https://github.com/tensorflow/tensorflow/pull/62591/commits/7c303884dfb707307f55d858fe5aec02e33d809a).
I hope this meets what you say.